### ISSUE TITLE
War for Lormark/Upper Aedirn needs work

### ISSUE THREAD

**Issue by ajjr1996** [ajjr1996/Witcher#8]
Saskia needs to be killed or we need an event where she dies and you get an event saying she flew away or something, most of all it's just not right to have saskia as the head of Upper Aedirn when Henselt/Stennis declare war to return it to the nobility and she needs to lost her titles if she loses the war.

**Comment by rquinio** [comment]
Reported on moddb: If Saskia wins the war against Henselt, he becomes her liege and upper aedirn becomes part of kaedwen.

**Comment by ajjr1996** [comment]
This needs done more than anything.

**Comment by rquinio** [comment]
Could not find the cause of the issue with Saskia becoming a vassal of Kaedwen when the war ends (both in case of invalidation or white peace).
It also happens when changing the war cb to vanilla claim CB, so it's not something with the claim_lomark cb...
Looks like an issue with the engine, but not sure what the conditions exactly are.

**Comment by ajjr1996** [comment]
Looks like I'm having some problems with this war, it seems as though the outcomes are bugged. If Stennis and Saskia try to bribe into peace Henselt and he accepts it he never gets any money. Also, If Henselt wins he has the option to take Saskia's remaining counties Tiel and Helligt but the event doesn't work and nothing happens. Lastly, when peace is agreed upon on both sides Upper Aedirn is momentarily vassalized by kaedwen, renamed Lormark for a day or so, then released and renamed back to Upper Aedirn. I'm not sure what is causing it.

**Comment by rquinio** [comment]
The renaming is based on the culture of the ruler (cultural name in landed titles).

**Comment by ajjr1996** [comment]
Yeah I know what the renaming is based on.. That's not really what I was getting at.
